src/gluonts/mx/distribution/mixture.py

```diff
@@ -122,10 +122,21 @@ def cdf(self, x: Tensor) -> Tensor:
     @property
     def stddev(self) -> Tensor:
         F = self.F
-        stddev_values = F.stack(*[c.stddev for c in self.components], axis=-1)
-        return F.sum(
-            F.broadcast_mul(stddev_values, self.mixture_probs, axis=-1),
-            axis=-1,
+        sq_mean_values = F.square(
+            F.stack(*[c.mean for c in self.components], axis=-1)
+        )
+        sq_std_values = F.square(
+            F.stack(*[c.stddev for c in self.components], axis=-1)
+        )
+
+        return F.sqrt(
+            F.sum(
+                F.broadcast_mul(
+                    sq_mean_values + sq_std_values, self.mixture_probs, axis=-1
+                ),
+                axis=-1,
+            )
+            - F.square(self.mean)
```

Comment:
I think this will work for the univariate case only, but it's fine like this for now and we can open a separate issue about further improving it